test/unittests/commands/ingest_xmlupload/test_apply_ingest_id.py

```diff
@@ -0,0 +1,61 @@
+from lxml import etree
+
+from dsp_tools.commands.ingest_xmlupload.apply_ingest_id import replace_filepath_with_sipi_id
+
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+
+class TestReplaceBitstreamPaths:
+    def test_all_good(self) -> None:
+        xml = (
+            "<knora>"
+            '<resource label="Fluffy Image" restype=":Image2D" id="Fluffy1" permissions="res-default">'
+            '<bitstream permissions="prop-default">images/Fluffy.jpg</bitstream>'
+            "</resource>"
+            "</knora>"
+        )
+        root = etree.ElementTree(etree.fromstring(xml))
+        reference_dict = {"images/Fluffy.jpg": "fluffy_id"}
+        res_tree, res_msg = replace_filepath_with_sipi_id(root, reference_dict)
```

Comment:
The variable name `res_msg` is subpar. This is not a message (=str), but an `IngestInformation` object, so what about `ingest_info`?